docker/main.go

```diff
@@ -46,7 +46,7 @@ func (repository *Repository) Login(username, password string) {
 
 func (repository *Repository) Build(tag string) {
 	console.Debug("Building Docker image [%s]", repository.UriFor(tag))
-	console.Shell("docker build --tag %s .", repository.UriFor(tag))
+	console.Shell("docker build --rm=false --tag %s .", repository.UriFor(tag))
```

Comment:
Hmm - this seems unrelated to the fix proposed. Happy to merge it separately with some explanation about the benefits.

Reply:
Yes, I will put it into separate branch. Shortens build time when you frequently rebuild the same image.